perf/lua/uri_escape_unescape.lua

```diff
@@ -24,23 +31,37 @@ local decode_sample = uri.escape(encode_sample, escape_opts)
 -- iterations slower than next ones. To avoid such effect we can introduce
 -- "warmup" iterations, disable JIT at all or make JIT compilation
 -- aggressive from the beginning.
---   - hotloop=1 enables compilation in loops on first iteration.
---   - hotexit=1 enables faster compilation for root traces as well as side
---     traces.
+--   - hotloop=1 enables compilation in loops after the first iteration.
```

Comment:
it is not related to commit, I would move it to a separate commit

Reply:
Splitted to the separate commit, thanks!